reana_workflow_controller/rest/workflows.py

```diff
@@ -217,6 +225,8 @@ def get_workflows(paginate=None):  # noqa
         sort = request.args.get("sort", "desc")
         search = request.args.get("search", "")
         status_list = request.args.get("status", "")
+        include_progress = request.args.get("include_progress", "")
+        include_workspace_size = request.args.get("include_workspace_size", "")
```

Comment:
Here it'd be safer to have `False` as the default value instead of an empty string.